src/operator-manual/monitor-mode/01-intro.md

```diff
@@ -0,0 +1,91 @@
+# Monitor mode
+
+When a policy is deployed on the cluster, it will start accepting,
+rejecting or mutating requests. It's a binary operation, either the
+policy is taking decisions on operations it targets, or it's not
+installed at all.
+
+In order to let users deploy a policy to a cluster, without
+letting it take final decisions, the `monitor` mode has been included.
+
+The `monitor` mode is a way to deploy policies to the cluster in a way
+that:
+
+1. All requests that go through the policy will be accepted, as if the
+   policy didn't exist.
+1. The policy result is traced in the `policy-server`
```

Comment:
Numbering nit

```suggestion
2. The policy result is traced in the `policy-server`
```

Reply:
I usually take advantage of that working as expected when rendered, so big lists are still indented the same (>= 10), and take that as a common practice.

I accept your suggestion though :)